integration-test/src/test/resources/trino-ci-testset/testsets/lakehouse-iceberg/00006_datatype.sql

```diff
@@ -30,8 +30,8 @@ VALUES (NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,
 select * from tb01 order by f1;
 
 CREATE TABLE tb02 (
-    f1 VARCHAR(200) not null ,
-    f2 CHAR(20) not null ,
+    f1 VARCHAR not null ,
+    f2 VARCHAR not null ,
```

Comment:
The fields of f1 and f2 are the same. Can you remove f2?

Reply:
done